src/member/member.ts

```diff
@@ -52,22 +53,53 @@ export const buildMemberExtra = (extra: Partial<MemberExtra>) => ({
   ...extra,
 });
 
-export type Member = {
+export type Account = {
   id: UUID;
   name: string;
-  email: string;
 };
 
-export type CompleteMember = Member & {
+type AccountTypeProperty = {
   type: `${MemberType}` | MemberType;
+};
+
+export type Member = Account & {
+  email: string;
+};
+
+export type Guest = Account;
+
+export type AugmentedAccount =
```

Comment:
I'm not a big fan of this name. In my point of view, it doesn't reflect the nature of this type. When I have read `AugmentedAccount` I thought it was an account with more features, but it is a Member or a guest user.

I don't like my proposition, but having something like `MemberOrGuest` allows to directly understand the nature of this type. If it's to give the access on an item to a guest or a member, we should try to reflect this purpose directly in the name if possible.

I don't know if I'm the only one who didn't like this name and if you have other suggestions ?